### DISCUSSION TITLE
能不能增加个功能。可以下载到群辉本地目录。

### DISCUSSION THREAD

**Discussion by NerRoshwin** [SPlayer-Dev/SPlayer#192] [Q&A]
### 是网页端还是客户端

群辉

### 当前系统环境

docker

### 当前 Node.js 及 npm 版本

v18

### 当前版本

v1.0.0

### 具体信息

不知道怎么改代码。 能不能增加个功能就是下载到docker映射目录。我群辉本地串流。希望能直接下载到nas里面。感谢大佬的应用。

**Comment by SettingDust** [comment]
可以 syncthing 或者其他同步方案

**Comment by NerRoshwin** [comment]
感谢大佬。感觉既然映射docker地址。直接下载到群辉本地。功能很好实现呀。大佬回头更新的时候考虑下哈。

从 Windows 版邮件发送

发件人: SettingDust
发送时间: 2024年6月16日 11:32
收件人: imsyy/SPlayer
抄送: NerRoshwin; Author
主题: Re: [imsyy/SPlayer] 能不能增加个功能。可以下载到群辉本地目录。 (Issue #173)

可以 syncthing 或者其他同步方案
